### PR TITLE
Use float pixel ratio

### PR DESCRIPTION
**Pull request by WhoAteDaCake**
Will prevent division by 0 crash as mentioned in #180.

### CODE REVIEW

**Comment by WhoAteDaCake** [comment]
Not quite sure if `NodeDrawContext` should also take float, that might mean some extra work

**Comment by WhoAteDaCake** [comment]
Thanks @OhadRau I've updated the tests

File: src/UI/TextNode.re [hunk 1]
```diff
@@ -117,7 +117,7 @@ class textNode (text: string) = {
       /* TODO: Cache font locally in variable */
       let style = _super#getStyle();
       let font =
-        FontCache.load(style.fontFamily, style.fontSize * pixelRatio);
+        FontCache.load(style.fontFamily, int_of_float(float_of_int(style.fontSize) *. pixelRatio));
```
Comment (bryphe) [inline src/UI/TextNode.re]: Nice catch here!

**Comment by bryphe** [comment]
Merging now, thanks for the contribution @WhoAteDaCake !

**Pull request merged**